client/task-list/tasks/payments/paypal.js

```diff
@@ -1,118 +1,151 @@
+/* global ppcp_onboarding */
 /**
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
 import apiFetch from '@wordpress/api-fetch';
-import { Button, CheckboxControl } from '@wordpress/components';
-import { Component, Fragment } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import { Component, Fragment, useEffect } from '@wordpress/element';
 import { compose } from '@wordpress/compose';
 import interpolateComponents from 'interpolate-components';
 import { withDispatch, withSelect } from '@wordpress/data';
 import { isEmail } from '@wordpress/url';
-import { Form, Link, Stepper, TextControl } from '@woocommerce/components';
+import { Form, Link, TextControl, Stepper } from '@woocommerce/components';
 import { getQuery } from '@woocommerce/navigation';
-import {
-	PLUGINS_STORE_NAME,
-	OPTIONS_STORE_NAME,
-	WC_ADMIN_NAMESPACE,
-} from '@woocommerce/data';
+import { PLUGINS_STORE_NAME, OPTIONS_STORE_NAME } from '@woocommerce/data';
 
-export class PayPal extends Component {
+const PAYPAL_PLUGIN = 'woocommerce-paypal-payments';
+const WC_PAYPAL_NAMESPACE = '/wc-paypal/v1';
+
+function loadOnboardingScript( url, data, onLoad ) {
```

Comment:
Loads onboarding script needed, and comes from the newly enabled plugin (that's why we add it on the fly).